WordPress/src/main/java/org/wordpress/android/sharedlogin/provider/SharedLoginProvider.kt

```diff
@@ -28,6 +28,9 @@ class SharedLoginProvider : QueryContentProvider() {
         sortOrder: String?
     ): Cursor? {
         inject()
+        if (!jetpackProviderSyncFeatureConfig.isEnabled()) {
```

Comment:
Actually, after solving the conflict like I explained in the PR description (see https://github.com/wordpress-mobile/WordPress-Android/pull/17320/commits/6907721d1899281560bc844193a38446fa6cf856) — which got rid of the `import org.wordpress.android.util.config.JetpackProviderSyncFeatureConfig` — that line no longer compiles and makes CI goes red with:
```
e: /var/lib/buildkite-agent/builds/ci-android-i-00464ed7936fef979-1/automattic/wordpress-android/WordPress/src/main/java/org/wordpress/android/sharedlogin/provider/SharedLoginProvider.kt: (31, 14): Unresolved reference: jetpackProviderSyncFeatureConfig
```

Which means that my conflict resolution was probably wrong after all.

@RenanLukas can I let you fix this and redo the conflict resolution properly, since you're the one who worked on those lines of code? Thanks!

Reply:
Hey @AliSoftware 👋 

Thanks for looking into this conflict. `ClientVerification` replaces what we've been doing with `JetpackPublicData`, so removing `JetpackPublicData` is correct.

Looks like what's missing in `SharedLoginProvider` is the `JetpackProviderSyncFeatureConfig` import and also the injected variable, so basically this line with the other imports:

```
import org.wordpress.android.util.config.JetpackProviderSyncFeatureConfig
```

And this line with the other injected variables:
```
@Inject lateinit var jetpackProviderSyncFeatureConfig: JetpackProviderSyncFeatureConfig
```

How should we proceed in this branch? Should I submit a PR targeting this branch adding these two lines?

Thanks!

Reply:
@RenanLukas thanks for taking a look!

Feel free to directly push a new commit to fix the issue directly on this PR. This PR is already using `merge/20.9-rc-3` as the head branch (and not `release/20.9`) as I already created and cut that `merge` branch from the release branch exactly in order to solve the conflict within that intermediate branch without risking impacting the release branch itself while solving the conflict.

So everything is already set for you to continue push more commits on that `merge/20.9-rc-3` branch to finish solving the conflict resolution properly, no need for an separate PR 🙂

Reply:
[Done](https://github.com/wordpress-mobile/WordPress-Android/pull/17320/commits/076befb3c417c7f1fc3232e84837ae9ffba202b7) 👍

Reply:
Thanks! 🎉